content/en/docs/languages/php/context.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Context
+weight: 55
+cSpell:ignore: Swoole
+---
+
+In order for OpenTelemetry to work, it must store and propagate important
+telemetry data. For example, when a request is received and a span is started it
+must be available to a component which creates its child span. To solve this
+problem, OpenTelemetry stores the span in the Context.
+
+More information:
+
+- [Context specification](/docs/specs/otel/context/)
+
+## Context
+
+Context is globally available, and there can only be one
+[active context](#active-context) in the current execution context. Context can
+store values (for example, a `Span`).
```

Comment:
```suggestion
Context is globally available, and there can only be one
[active context](#active-context) in the current execution context.
Context can store values (for example, a `Span`).
```

"globally" refers to PHP application scope? OTel scope?

Reply:
PHP application scope

Reply:
updated to clarify that it's globally available within a single php execution context